geostructures/geohash.py

```diff
@@ -391,23 +392,26 @@ def hash_collection(
         Keyword Args:
             resolution:
                 The H3 resolution to apply
+            agg_fn:
+                A function that accepts a list of geoshapes

```

Comment:
Quickly describe the default behavior

Reply:
Here instead of in the first docstring paragraph, or in addition?

Reply:
I'd say instead of? Either works for me though